src/components/ChecAccordion.vue

```diff
@@ -136,25 +161,22 @@ export default {
   &__body-container {
     @apply max-h-0 overflow-hidden mb-4;
 
-    transition: max-height 700ms cubic-bezier(0, 1, 0, 1),
-      margin-bottom 100ms linear 200ms;
+    transition: max-height 300ms cubic-bezier(0, 1, 0, 1),
+      margin-bottom 300ms cubic-bezier(0, 1, 0, 1);
   }
 
   &__body {
-    @apply p-4 pb-0;
+    @apply p-4;
 
-    transition: padding-bottom 100ms linear 200ms;
+    //transition: padding-bottom 100ms linear 200ms;
```

Comment:
```suggestion
```

And previous line (surprised this isn't a linting error)

Reply:
I guess it doesn't know that the content of the comment is useless.